modules/ROOT/pages/raise-error-component-reference.adoc

```diff
@@ -4,22 +4,16 @@ include::_attributes.adoc[]
 endif::[]
 :keywords: mule, esb, studio, raise, error
 
-This Core component generates a xref:mule-error-concept.adoc[Mule Error], as if a failure had occurred, which allows you to customize its description and type.
+This Core component generates a Mule Error, as if a failure had occurred, which allows you to customize its description and type.
 
-Note that you can only use core types, such as `SECURITY` or custom ones. You cannot raise an
-error from another module like `FILE:FILE_NOT_FOUND`.
+Note that you only this component to raise core runtime errors, such as `MULE:SECURITY`, `MULE:CONNECTIVITY`, etc, or custom ones.
```

Comment:
`Note that you only this component to raise core runtime errors` -> `Note that you only use this component to raise core runtime errors`?